azurerm/resource_arm_scheduler_job_collection_test.go

```diff
@@ -0,0 +1,135 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMSchedulerJobCollection_basic(t *testing.T) {
+	ri := acctest.RandInt()
+	resourceName := "azurerm_scheduler_job_collection.test"
+	config := testAccAzureRMSchedulerJobCollection(ri, testLocation(), "")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMSchedulerJobCollectionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMSchedulerJobCollectionExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMSchedulerJobCollection_complete(t *testing.T) {
+	ri := acctest.RandInt()
+	resourceName := "azurerm_scheduler_job_collection.test"
+	config := testAccAzureRMSchedulerJobCollection(ri, testLocation(), `
+  state = "disabled"
+  quota {
+    max_recurrence_frequency = "hour"
+    max_retry_interval       = 10
+    max_job_count            = 10
+  }
```

Comment:
can we separate this out into a separate configuration called `testAccAzureRMSchedulerJobCollection_complete` to match the other tests?